app/parsers/bulkrax/bagit_parser.rb

```diff
@@ -122,18 +123,27 @@ def current_record_ids
     def write_files
       require 'open-uri'
       require 'socket'
+
```

Comment:
what the new file structure looks like when bags are being created for export
![image](https://user-images.githubusercontent.com/29032869/177659304-cb224312-f15a-4098-a7c9-41d7d3b88ad2.png)